commands/publish/index.js

```diff
@@ -62,7 +63,10 @@ class PublishCommand extends Command {
     }
 
     if (this.options.requireScripts) {
-      this.logger.info("require-scripts", "enabled");
+      throw new ValidationError(
+        "removed",
+        "--require-scripts has been removed.\nExecute these scripts from the appropriate package lifecycle."
```

Comment:
Hopefully this is clear enough?

<img width="1092" alt="screen shot 2019-01-11 at 09 50 18" src="https://user-images.githubusercontent.com/5605/51050637-578a6780-1586-11e9-82b1-f3f9a52f4cae.png">

Reply:
If you want to make it clearer, you could add a link to https://docs.npmjs.com/misc/scripts

Reply:
Done!